### PR TITLE
Added column wrappers around grid slots and added container query name

### PR DESCRIPTION
**Pull request by bspeare**

### CODE REVIEW

File: src/components/grid/grid.scss [hunk 1]
```diff
@@ -18,3 +18,12 @@
     }
   }
 }
+
+// @todo move this so a file that only storybook uses?
```
Comment (bspeare) [inline src/components/grid/grid.scss]:
```suggestion
// @todo move this to a file that only storybook uses?
```

File: src/components/grid/grid.scss [hunk 2]
```diff
@@ -18,3 +18,12 @@
     }
   }
 }
+
+// @todo move this to a file that only storybook uses?
```
Comment (bspeare) [inline src/components/grid/grid.scss]:
```suggestion
```

**Comment by pyrello** [comment]
I was trying to get something like this working: https://michaelnthiessen.com/supercharge-your-slots/#adding-slots-dynamically. I wasn't able to figure it out and it isn't worth spending more time on for now, but I want to come back to it eventually.

**Review by pyrello — APPROVED** [review]
This should be good to get us started.

**Pull request merged**